product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.1-converting-soap-to-json/1.1.1-soap-to-json-using-payloadfactory-mediator/src/test/java/org/wso2/carbon/esb/endpoint/test/AuthenticatorClient.java

```diff
@@ -0,0 +1,82 @@
+/*
+ *Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *WSO2 Inc. licenses this file to you under the Apache License,
+ *Version 2.0 (the "License"); you may not use this file except
+ *in compliance with the License.
+ *You may obtain a copy of the License at
+ *
+ *http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *Unless required by applicable law or agreed to in writing,
+ *software distributed under the License is distributed on an
+ *"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *KIND, either express or implied.  See the License for the
+ *specific language governing permissions and limitations
+ *under the License.
+ */
+
+package org.wso2.carbon.esb.endpoint.test;
+
+import org.apache.axis2.client.Stub;
+import org.apache.axis2.context.ServiceContext;
+import org.apache.axis2.transport.http.HTTPConstants;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.authenticator.stub.AuthenticationAdminStub;
+
+public class AuthenticatorClient {
+
+    private static final Log log = LogFactory.getLog(AuthenticatorClient.class);
+    private AuthenticationAdminStub authenticationAdminStub;
+
+    public AuthenticatorClient(String backendUrl) throws Exception {
```

Comment:
Shall we add method comments to all the methods on this class?

Reply:
Fixed by PR [https://github.com/wso2/product-ei/pull/3471](https://github.com/wso2/product-ei/pull/3471)